test/integration/Cover/expireCover.js

```diff
@@ -0,0 +1,612 @@
+    await ybETH.connect(tokenOwner).transfer(coverBuyer1.address, parseEther('100'));
+    await ybETH.connect(coverBuyer1).approve(yc.address, parseEther('100'));
+  }
+
+  async function buyCover({
+    amount,
+    targetPrice,
+    priceDenominator,
+    productId,
+    coverAsset,
+    period,
+    cover,
+    coverBuyer1,
+  }) {
+    const expectedPremium = amount.mul(targetPrice).div(priceDenominator);
+
+    await cover.connect(coverBuyer1).buyCover(
+      {
+        owner: coverBuyer1.address,
+        productId,
+        coverAsset,
+        amount,
+        period,
+        maxPremiumInAsset: expectedPremium,
+        paymentAsset: coverAsset,
+        payWitNXM: false,
+        commissionRatio: parseEther('0'),
+        commissionDestination: AddressZero,
+        ipfsData: '',
+      },
+      [{ poolId: '0', coverAmountInAsset: amount.toString() }],
+      {
+        value: expectedPremium,
+      },
+    );
+  }
+
+  async function submitIncident({ yc, productId, period, signer }) {
+    // submit incident
+    const { timestamp: currentTime } = await ethers.provider.getBlock('latest');
+
+    await yc
+      .connect(signer)
+      .submitIncident(productId, parseEther('1.1'), currentTime + period / 2, parseEther('100'), '');
+  }
+
+  it('expire a cover that had a claim paid out fully', async function () {
+    const { DEFAULT_PRODUCT_INITIALIZATION } = this;
+    const { cover, stakingPool0, as, yc, ybETH } = this.contracts;
+    const [coverBuyer1, staker1] = this.accounts.members;
+    const [nonMember1] = this.accounts.nonMembers;
+    const { emergencyAdmin } = this.accounts;
+
+    const { productId, coverAsset, period, gracePeriod, amount, coverId, segmentId } = expireCoverFixture;
+
+    // Stake to open up capacity
+    await stake({ stakingPool: stakingPool0, staker: staker1, productId, period, gracePeriod });
+
+    // cover buyer gets yield token
+    await transferYieldToken({ tokenOwner: this.accounts.defaultSender, coverBuyer1, yc, ybETH });
+
+    // enable active cover tracking
+    await cover.connect(emergencyAdmin).enableActiveCoverAmountTracking([], []);
+    await cover.connect(emergencyAdmin).commitActiveCoverAmounts();
+
+    // Buy Cover
+    await buyCover({
+      ...expireCoverFixture,
+      targetPrice: DEFAULT_PRODUCT_INITIALIZATION[0].targetPrice,
+      cover,
+      coverBuyer1,
+    });
+
+    // submit incident
+    await submitIncident({ yc, productId, period, signer: this.accounts.defaultSender });
+
+    // accept incident
+    await as.connect(staker1).castVotes([0], [true], parseEther('100'));
+
+    {
+      // advance past payout cooldown
+      const { payoutCooldownInDays } = await as.config();
+
+      const { end } = await as.getPoll(0);
+      await setTime(end + daysToSeconds(payoutCooldownInDays));
+    }
+
+    // fully paid cover
+    await yc.connect(coverBuyer1).redeemPayout(0, 0, 0, amount, nonMember1.address, []);
```

Comment:
let's use name constants here for clarity 


incidentId
coverId
segmentId

use it above in asessmentIds for castVotes as well.